walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,66 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+import java.util.*;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    List<TimeRange> unavailable_Time = new ArrayList<TimeRange>();
```

Comment:
Here and elsewhere: we should follow a consistent style of using camelcase for variable names. So unavailable_Time becomes unavailableTime. https://google.github.io/styleguide/javaguide.html#s5.1-identifier-names

Reply:
Ya, I usually do camel case, but at my internship that's what they do sometimes, so it was kind of in my head